dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -482,118 +282,6 @@ class RegistrationsControllerTest < ActionController::TestCase
     assert user.deleted?
   end
 
-  # The next several tests explore profile changes for users with or without
-  # passwords.  Examples of users without passwords are users that authenticate
-  # via oauth (a third-party account), or students with a picture password.
-
-  # Tech debt note:
-  # These tests make multiple controller calls per-test, which is not fully supported as per http://api.rubyonrails.org/v4.2/classes/ActionController/TestCase.html
-  # Currently this is worked around by calling current_user.reload in registrations_controller.rb, but ideally
-  # these tests should be fixed up to avoid this issue.
```

Comment:
This "Tech debt note" didn't move, because it no longer applies.  We don't have to call `current_user.reload` in `RegistrationsController#update` anymore since the relevant tests are not part of an `IntegrationTest`.